PubNubSwiftConsole/Classes/UI/CollectionView/Cells/SubscribeStatusCollectionViewCell.swift

```diff
@@ -86,6 +100,18 @@ class SubscribeStatusCollectionViewCell: CollectionViewCell {
         } else {
             timeTokenLabel.hidden = true
         }
+        if item.client.isSubscribingToChannels, let channelsLabelString = item.client.channelsString() {
+            channelLabel.hidden = false
```

Comment:
Use the changes from this PR to implement channels and channel group parsing https://github.com/pubnub/SwiftConsole/pull/30